src/plugins/intel_cpu/src/emitters/snippets/aarch64/jit_memory_emitters.cpp

```diff
@@ -88,19 +101,22 @@ void jit_load_broadcast_emitter::emit_isa(const std::vector<size_t> &in, const s
     h->uni_ld1rw(dst.s, src, byte_offset);
 }
 
-jit_store_memory_emitter::jit_store_memory_emitter(jit_generator* h, cpu_isa_t isa, const ExpressionPtr& expr) : jit_memory_emitter(h, isa, expr) {
-    OV_CPU_JIT_EMITTER_ASSERT(src_prc == dst_prc, "Only supports equal input and output types but gets ",
-                              src_prc.get_type_name(),
-                              " and ",
-                              dst_prc.get_type_name());
-    OV_CPU_JIT_EMITTER_ASSERT(src_prc == ov::element::f32, "Only supports FP32 precision.");
-
-    const auto store = ov::as_type_ptr<snippets::op::Store>(expr->get_node());
-    OV_CPU_JIT_EMITTER_ASSERT(store != nullptr, "Expects Store expression");
-    count = store->get_count();
-    byte_offset = store->get_offset();
-    in_out_type_ = emitter_in_out_map::vec_to_gpr;
-    store_emitter.reset(new jit_store_emitter(h, isa, src_prc, dst_prc, count, byte_offset));
+jit_store_memory_emitter::jit_store_memory_emitter(jit_generator* h, cpu_isa_t isa, const ExpressionPtr& expr)
+    : jit_memory_emitter(h, isa, expr, emitter_in_out_map::vec_to_gpr) {
+    OV_CPU_JIT_EMITTER_ASSERT(one_of(src_prc, ov::element::f32, ov::element::i32, ov::element::f16, ov::element::i8, ov::element::u8),
+                              "Unsupported input type: ", src_prc.get_type_name());
+    OV_CPU_JIT_EMITTER_ASSERT(one_of(dst_prc, ov::element::f32, ov::element::i32, ov::element::f16, ov::element::i8, ov::element::u8),
+                              "Unsupported output type: ", dst_prc.get_type_name());
```

Comment:
What's about Store op `fp16->u8`? As I can see, it's unsupported case 🤔 
Could you please update the validation checks for `Store` and `Load`?

Reply:
Good point! Applied. Thanks Alexandra!